plotly/offline.py

```diff
@@ -0,0 +1,149 @@
+""" Plotly Offline
+    A module to use Plotly's graphing library with Python
+    without connecting to a public or private plotly enterprise
+    server.
+"""
+from __future__ import absolute_import
+
+from plotly import utils
+import uuid
+import json
+import os
+import requests
+
+
+def download_plotlyjs(download_url, plotlyjs_dir='~/.plotly/plotlyjs'):
```

Comment:
Can we get rid of the `download_url` param and just create a streambed url that we *know* and can authenticate at?

Then, we can just write a function to decide whether the requesting user has access?

Additionally, why do we need the `plotlyjs_dir` arg? is there a good reason for folks to need to change this? Also, i think it's not used in the func?

(obviously, the download_url is perfect for the *current* dev though :))